config/xalign.yaml

```diff
@@ -47,3 +47,7 @@ crossExchangeStrategies:
       USDT: 100
       USDC: 100
       TWD: 3000
+    alertAmounts:
+      USDT: 200
+      USDC: 200
+      TWD: 6000
```

Comment:
we need to add a quoteCurrency config so that we can convert BTC, ETH into the quote currency

Reply:
and use a single field like `slackAlertQuoteAmount` as the threshold